forms.rst

```diff
@@ -562,8 +562,8 @@ the correct values of a number of field options.
     field ``nullable``). This is very useful, as your client-side validation will
     automatically match your validation rules.
 
-``max_length``
-    If the field is some sort of text field, then the ``max_length`` option can be
+``maxlength``
```

Comment:
Thanks for opening this PR! This change is legit, but I think you should also replace `option` by `html attribute` for both this one and `required` above. since this does not use the options at all, but the `attr` option, btw `max_length` option has been deprecated in 2.5.